Emotion/Windows/ClientLibrary/Contract/Scores.cs

```diff
@@ -75,6 +75,26 @@ public class Scores
         /// 
         /// </summary>
         public float Surprise { get; set; }
+        
```

Comment:
wow it looks really cool, It's better than my implementation because you can get all the emotions ordered. So are you going to write that function or do you want that I write that function? 
pd: because It's you're code, I will try to implement that for Android :)